Resources/public/snippets/common/navigation/navigationMain.less

```diff
@@ -17,13 +17,12 @@
 
           &.level_1 {
             height: 50px;
-            background: url("/bundles/chameleonsystemchameleonshoptheme/images/layout/mainnavi.png") repeat-x bottom center;
```

Comment:
The image files, that are not used anymore should be deleted.

Reply:
ok, done